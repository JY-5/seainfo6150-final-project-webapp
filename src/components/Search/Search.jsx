import React from 'react'
import styles from "./Search.module.css";

const Search = () => {
    return (
        <div>        
            <input className={styles.search} type="text" name="search"/>
            <input className={styles.button} type="submit" value="Search" />
        </div>
    )
}

export default Search