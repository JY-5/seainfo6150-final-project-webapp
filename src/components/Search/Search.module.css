.search {
    height: 1rem;
    margin-right: 1rem;
    width: 15rem;
    height: 1.5rem;
}

.button {
    font-size: 1rem;
    padding: 0.2rem;
}