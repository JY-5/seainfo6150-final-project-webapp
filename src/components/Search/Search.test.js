import React from "react";
import Search from './Search';
import { render } from "@testing-library/react";

describe("Add Search tests", () => {
    it("renders correctly", () => {
      const { container } = render(<Search/>);
      expect(container).toMatchSnapshot();
    });
});