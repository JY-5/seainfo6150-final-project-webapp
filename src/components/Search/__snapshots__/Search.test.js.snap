// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Search tests renders correctly 1`] = `
<div>
  <div>
    <input
      class="search"
      name="search"
      type="text"
    />
    <input
      class="button"
      type="submit"
      value="Search"
    />
  </div>
</div>
`;
